resource/subfunctions/started_task.md

```diff
@@ -0,0 +1,276 @@
+
+## `ResourceAdmin.add_started_task()`
+
+```python
+  def add_started_task( self, started_task_name: str, traits: dict = {} ) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+&nbsp;
+
+{: .experimental }
+> _Only a subset of available **Segments** and **Traits** are considered **Stable**. See [Segments](../../base/segments_traits_operators#segments) and [Traits](../../base/segments_traits_operators#traits) for more details._
+
+&nbsp;
+
+Define a new **Started Task** profile in the **STARTED** class.
+
+#### 📥 Parameters
+* `started_task_name`<br>
+  The name of the **Started Task** profile being defined to the **STARTED** class.
+
+* `traits`<br>
+  A dictionary of **traits/attributes** that should be given to the resource on creation. See [Traits](../../base/segments_traits_operators#traits) to see what all of the valid **Resource Traits** are.
+
+#### 📤 Returns
+* `Union[dict, bytes]`<br>
+  Returns a **Security Result dictionary** or **Security Request XML bytes** if the `ResourceAdmin.generate_requests_only` class attribute is set to `True`.
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `AddOperationError`<br>
+  Raises `AddOperationError` when the **general resource profile** cannot be added because it already exists.
+* `SegmentTraitError`<br>
+  Raises `SegmentTraitError` when the dictionary of **traits/attributes** provided contains one or more **unknown** traits.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+from pyracf import ResourceAdmin
+resource_admin = ResourceAdmin()
+resource_admin.add_started_task("TSTTSKEL")
+{'securityResult': {'resource': {'name': 'TSTTSKEL', 'class': 'STARTED', 'operation': 'set', 'requestId': 'ResourceRequest', 'commands': [{'safReturnCode': 0, 'returnCode': 0, 'reasonCode': 0, 'image': 'RDEFINE STARTED             (TSTTSKEL) ', 'messages': ['ICH10006I RACLISTED PROFILES FOR STARTED WILL NOT REFLECT THE ADDITION(S) UNTIL A SETROPTS REFRESH IS ISSUED.']}]}, 'returnCode': 0, 'reasonCode': 0}}
+```
+
+###### Security Result Dictionary as JSON
+```json
+{
+  "securityResult":{
+    "resource":{
+      "name":"TSTTSKEL",
+      "class":"STARTED",
+      "operation":"set",
+      "requestId":"ResourceRequest",
+      "commands":[
+        {
+          "safReturnCode":0,
+          "returnCode":0,
+          "reasonCode":0,
+          "image":"RDEFINE STARTED             (TSTTSKEL) ",
+          "messages":[
+            "ICH10006I RACLISTED PROFILES FOR STARTED WILL NOT REFLECT THE ADDITION(S) UNTIL A SETROPTS REFRESH IS ISSUED."
+          ]
+        }
+      ]
+    },
+    "returnCode":0,
+    "reasonCode":0
+  }
+}
+```
+
+## `ResourceAdmin.alter_started_task()`
+
+```python
+  def alter_started_task( self, started_task_name: str, traits: dict = {} ) -> Union[dict, bytes]:
+```

```

Comment:
```python
def alter_started_task(self, started_task_name: str, traits: dict = {}) -> Union[dict, bytes]:
```

Reply:
Leading spaces and unnecessary trailing spaces and indents removed across all function definitions.